app-misc/peaclock/peaclock-0.1.3.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A colourful binary clock for the terminal"
+DESCRIPTION="Features:
+* digital clock
+* binary clock
+* 12/24 hour time
+* customize with hex colour codes
+* set a custom unicode character for the binary clock graphic
+* compact or expanded mode
+* toggle visibility of digital and binary clocks
+* command prompt with a readline-like interface"
+HOMEPAGE="https://github.com/octobanana/peaclock"
+COPYRIGHT="2019 Brett Robinson"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0041de0e50429786dff4152c60f6dbd84249287d289ad6d7573e47c9f9b62afc"
+SOURCE_FILENAME="peaclock-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [[ $targetArchitecture == x86_gcc2 ]]; then
```

Comment:
```suggestion
if [ "$targetArchitecture" = x86_gcc2 ]; then
```

Currently most of our recipes are using the sh version of this condition, so I think we should stick to that.

Reply:
Oh, I thought that it would be fine since HaikuPorter seems to require and use Bash. So should only POSIX `sh` features be used in recipes or is this just because they're nearly equivalent?

Reply:
Yes, we use bash, but @fbrosson tries to avoid bash-specific construct and established that as an unofficial convention across HaikuPorts sometimes ago. The reasoning is in some merged PRs long ago IIRC.

Reply:
Where is the line drawn between when it is acceptable to use Bashisms and not? Is it when POSIX has no equivalent feature? Also, is it fine to go all out with Bashisms when a recipe already uses some?